jsonnet/components/kube-state-metrics.libsonnet

```diff
@@ -201,9 +201,9 @@ function(params)
                         |||
                           --metric-denylist=
                           ^kube_secret_labels$,
-                          ^kube_.+_annotations$
```

Comment:
This will not have any impact once o/ksm is bumped to v2.10.0 since we remove these metrics by default now: https://github.com/kubernetes/kube-state-metrics/releases/tag/v2.10.0